common/culture/cultures/trogg.txt

```diff
@@ -133,5 +133,56 @@
 			{ name = "mercenary_company_band_of_horned_welshmen" }
 		}
 	}
+
+	drogbar = {
+		graphical_cultures = {
+			tibetan_group_coa_gfx
+			indian_building_gfx
+			mongol_clothing_gfx
+			mongol_unit_gfx
+			
+			drogbar_portrait_gfx
+		}
+		color = rgb { 0 176 250 }
+
+		cadet_dynasty_names = {
+			"dynn_Elwynn"
+		}
+
+		dynasty_names = {
+			"dynn_Wrynn"
+		}
```

Comment:
I would invent dynasty names to avoid doing it in the future.
I suggest using the Eastern pattern: the name of the founder + **id**. Example: **Amgrolid**.